src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi4Driver.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace System.Device.Gpio.Drivers
+{
+    public unsafe partial class RaspberryPi4Driver : RaspberryPi3Driver
```

Comment:
is this file actually needed? Not sure what's the value in having empty class

Reply:
I mean we can leave it for compat but then I'd change RaspberryPi3Driver to be the same (and not partial) and create GenericRaspberryPiDriver or something similar (or ..Pi3Or4Driver)

Reply:
You mean rename the current RaspberryPi3Driver to GenericRaspberryPiDriver and then create 2 derived classes?

Reply:
Done.